src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/TaskSteps.java

```diff
@@ -33,114 +37,125 @@
 
     @Autowired
     private Tasks tasksPanel;
+    @Autowired
+    private SoftAssert softAssert;
```

Comment:
Could you update this PR according to the things that we already talked before?

Reply:
`    /**
     * Based on tag annotation enable soft assert.
     */
    @Before("@SoftAssert")
    public static void initialize() {
        softAssert = new SoftAssert();
    }

    /**
     * Final step validation for soft assert.
     */
    @And("asserts all")
    public static void assertAll() {
        softAssert.assertAll();
    }`